README.rst

```diff
@@ -242,8 +247,16 @@ should have the format ``_matrix._tcp.<yourdomain.com> <ttl> IN SRV 10 0 <port>
 Note that the server hostname cannot be an alias (CNAME record): it has to point
 directly to the server hosting the synapse instance.
 
-You can then configure your homeserver to use ``<yourdomain.com>`` as the domain in
-its user-ids, by setting ``server_name``::
+If it is not possible to acquire a certificate valid for you matrix domain,
+the correct method is create a .well-known entry. For more details on how to
+configure .well-known `see <https://github.com/matrix-org/synapse/blob/master/docs/well_known.md`_
+
+For those of you upgrading your TLS certificate in readiness for Synapse 1.0,
+please take a look at `our guide <https://github.com/matrix-org/synapse/blob/master/docs/MSC1711_certificates_FAQ.md#configuring-certificates-for-compatibility-with-synapse-100>`_
+
+Having created a valid TLS certificate you can then configure your homeserver
```

Comment:
In practice we now encourage people to create their configs up-front, and then tweak them later to enable TLS and federation. This is therefore backwards.